go.mod

```diff
@@ -22,6 +22,15 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
+	github.com/fatih/color v1.7.0 // indirect
+	github.com/hashicorp/go-hclog v0.14.1 // indirect
+	github.com/hashicorp/yamux v0.0.0-20180604194846-3520598351bb // indirect
+	github.com/mattn/go-colorable v0.1.4 // indirect
+	github.com/mitchellh/go-testing-interface v0.0.0-20171004221916-a61a99592b77 // indirect
+	github.com/oklog/run v1.0.0 // indirect
+)
+
 require (
```

Comment:
Can we merge these `require` blocks?

Reply:
Ooops, didn't pay enough attention since those were auto-generated. Updated.